src/server/req_rescq.c

```diff
@@ -773,11 +776,15 @@ req_confirmresv(struct batch_request *preq)
 	}
 	rc = assign_resv_resc(presv, next_execvnode, FALSE);
 
-	if (presv->ri_qs.ri_stime < time_now) {
+	DBPRT(("resv_name=%s, rc=%d, is_degraded=%d, stime=%ld, now=%ld",
+	       presv->ri_qs.ri_resvID, is_degraded, rc, presv->ri_qs.ri_stime, time_now));
```

Comment:
The order of `is_degraded` and `rc` is wrong.

Reply:
is_degraded is set at the top of the function and not changed.  You don't have to print it a second time.